main.go

```diff
@@ -16,7 +41,9 @@ type ClientConfig struct {
 	APISecret string // User/Client secret to use when communicating with the e3db API
 	// Hostname for the soon to be deprecated (v1) e3db bearer auth service API.
 	// Once request signing is the primary mode of authenticating e3db requests this can be removed.
-	AuthNHost string
+	AuthNHost      string
+	SigningKeys    SigningKeys    // AsymmetricEncryptionKeypair used for signing and authenticating requests
+	EncryptionKeys EncryptionKeys // AsymmetricEncryptionKeypair used for encrypting and decrypting data
```

Comment:
Does a request client need encryption keys?

Reply:
No, encryption keys are used for client side operations such as encrypting the record before writing(not currently implemented) and writing access keys for other users